README.md

```diff
@@ -551,6 +587,30 @@ services.Configure<TargetingEvaluationOptions>(options =>
 });
 ```
 
+## Targeting Exclusion
+
+When defining an Audience, users and groups can be excluded from the audience. This is useful when a feature is being rolled out to a group of users, but a few users or groups need to be excluded from the rollout. Exclusion is defined by adding a list of users and groups to the `Exclusion` property of the audience.
+```
+"Audience": {
+    "Users": [
+        "Jeff",
+        "Alicia"
+    ],
+    "Groups": [
+        {
+            "Name": "Ring0",
+            "RolloutPercentage": 100
+        }
+    ],
+    "DefaultRolloutPercentage": 0
+    "Exclusion": {
+        "Users": [
+            "Mark"
+        ]
+    }
+}
+```

```

Comment:
Can you add a short example of what the filter will evaluate to for different users?